website/app/assets/stylesheets/components/_feedback.scss

```diff
@@ -16,5 +16,7 @@
 .feedback-prompt__body {
   @include cads-typographic-scale-text-small();
 
-  margin: 0;
```

Comment:
if this is idiomatic Sass now to silence deprecation warnings, I hate it! 🙃

Reply:
It's specific to https://sass-lang.com/documentation/breaking-changes/mixed-decls/ to "opt in" to the new behaviour. I hate it too, alternative is to silence the warning with `--silence-deprecation=mixed-decls`. Not really sure which is least-worst.